packages/api-client/src/granules.ts

```diff
@@ -48,12 +49,7 @@ export const getGranuleResponse = async (params: {
     callback = invokeApi,
   } = params;
 
-  let path = `/granules/${collectionId}/${granuleId}`;
-
-  // Fetching a granule without a collectionId is supported but deprecated
-  if (!collectionId) {
-    path = `/granules/${granuleId}`;
-  }
```

Comment:
We'll need to strategize on this release/merge. The dashboard currently doesn't support querying granules by collectionId + granuleId.

Reply:
I think it's a more fair question to ask if this is a change we should make here and now given we still have a legit endpoint.  See comments in the PR above: 

https://github.com/nasa/cumulus/pull/3384#discussion_r1202692099